cookbook/profiler/data_collector.rst

```diff
@@ -203,7 +203,7 @@ must also define additional blocks:
             </div>
         {% endset %}
 
-        {{ include('@WebProfiler/Profiler/toolbar_item.html.twig') }}
+        {{ include('@WebProfiler/Profiler/toolbar_item.html.twig', {'link': profiler_url}) }}
```

Comment:
I think we should use either `{ link: true }` or `{ link: false }` here. Using `profiler_url` here looks like you could pass the actual URL here,